### DISCUSSION TITLE
Does conductor publish workflow events to SQS whenever there is a change in workflow status?

### DISCUSSION THREAD

**Discussion by SuryaSrikar** [Netflix/conductor#3716] [General]
Does the conductor publish workflow events to ActiveMQ/Kafka whenever there is a change in workflow status?

Usecase:
We want to update all other system to know the current status of the workflow.

**Comment by v1r3n** [comment]
Hi @SuryaSrikar This feature exists in Orkes hosted version of Conductor.

**Comment by JCHacking** [comment]
Hi @SuryaSrikar 

Not in base conductor, but with driver-community you have this option using queue publisher (I leave you a link to the readme of that specific module)
https://github.com/Netflix/conductor-community/tree/main/workflow-event-listener
